openfl-tutorials/interactive_api/MXNet_landmarks/README.md

```diff
@@ -0,0 +1,110 @@
+# MXNet Facial Keypoints Detection tutorial
+***Note: Please pay attention that this task uses the dataset from Kaggle. To get the dataset you
+will need a Kaggle account and accept "Facial Keypoints Detection" competition rules.***
+
+This tutorial shows how to use any other framework, different from already supported PyTorch and TensorFlow, together with OpenFl.
```

Comment:
Let's remove it from here since we already have this info below
```suggestion
This tutorial shows how to use any other framework, different from already supported PyTorch and TensorFlow, together with OpenFl.
```

Reply:
I think we should keep it, this note shortly explains why we should use Kaggle, the information below does not explain why we need it. In addition, I took it from the PyTorch_DogsCats_Vet tutorial to kеер description style for tutorials that use Kaggle.